opendbc/car/honda/hondacan.py

```diff
@@ -43,7 +43,8 @@ def get_lkas_cmd_bus(CAN, car_fingerprint, radar_disabled=False):
 
 def get_cruise_speed_conversion(car_fingerprint: str, is_metric: bool) -> float:
   # on certain cars, CRUISE_SPEED changes to imperial with car's unit setting
-  return CV.MPH_TO_MS if car_fingerprint in HONDA_BOSCH_RADARLESS and not is_metric else CV.KPH_TO_MS
+  # Todo: revisit this. i think i have old routes from canada-spec cars that worked differently than us-spec
+  return CV.MPH_TO_MS if car_fingerprint in (HONDA_BOSCH_RADARLESS | {CAR.HONDA_ODYSSEY_BOSCH}) and not is_metric else CV.KPH_TO_MS
```

Comment:
We can merge in dashcam with the TODO there, but make it clearer it's specific to Odyssey Bosch.